snoop/data/models.py

```diff
@@ -83,6 +83,17 @@ def create(cls):
         pk = fields.pop('sha3_256')
 
         Path(f.name).rename(BLOB_ROOT / pk)
+
+        if fields['mime_type'].startswith('text/'):
+            if looks_like_email(Path(BLOB_ROOT / pk)):
```

Comment:
Let's not repeat `Path(BLOB_ROOT / pk)`, that's liable to change pretty soon, if we want to store millions of blobs. At least save it into a variable.